File: app/src/main/java/com/molihua/hlbmerge/interfaces/IMainTitlebarFragment.java
package com.molihua.hlbmerge.interfaces;

import com.molihua.hlbmerge.fragment.impl.MainTitlebarFragment;
import com.xuexiang.xui.widget.searchview.MaterialSearchView;

public interface IMainTitlebarFragment {
    void setMainTitle(String text);
    
    void showHideImgView(boolean status);

    void showTitleImgView(MainTitlebarFragment.ImgView showImg);

    void showHideSearchView(boolean status);

    MaterialSearchView getSearchView();
}

File: app/src/main/java/com/molihua/hlbmerge/interfaces/IMainTitlebarFragment.kt
package com.molihua.hlbmerge.interfaces;

import com.molihua.hlbmerge.fragment.impl.MainTitlebarFragment;
import com.xuexiang.xui.widget.searchview.MaterialSearchView;

interface IMainTitlebarFragment {
    fun setMainTitle(text: String)

    fun showHideImgView(status: Boolean)

    fun showTitleImgView(showImg: MainTitlebarFragment.ImgView)

    fun showHideSearchView(status: Boolean)

    fun getSearchView(): MaterialSearchView
}
